charts/argocd-workflows-apps/values.yaml

```diff
@@ -0,0 +1,14 @@
+destination:
+  server: https://api.argus.diamond.ac.uk:6443
```

Comment:
It would probably be better to default to the `kubernetes.default.svc` domain, to make the value cluster portable